### PR TITLE
Dispose of XmlReaders

### PR DESCRIPTION
**Pull request by conniey**

### CODE REVIEW

File: src/Microsoft.Fx.Portability/TargetMapper.cs [hunk 1]
```diff
@@ -95,14 +95,18 @@ private void Load(Stream stream, string path)
...
 #endif
             };
 
+            XmlReader xmlReader = null;
+            XmlReader xmlSchemaReader = null;
+
             try
             {
-                var doc = XDocument.Load(XmlReader.Create(stream, readerSettings));
+                xmlReader = XmlReader.Create(stream, readerSettings);
```
Comment (twsouthwick) [inline src/Microsoft.Fx.Portability/TargetMapper.cs]: Why not combine with the declaration and use a `using`?

File: src/Microsoft.Fx.Portability/TargetMapper.cs [hunk 2]
```diff
@@ -115,9 +119,9 @@ private void Load(Stream stream, string path)
                         XmlResolver = null
                     };
 
-                    var reader = XmlReader.Create(xsdStream, xmlReaderSettings);
+                    xmlSchemaReader = XmlReader.Create(xsdStream, xmlReaderSettings);
```
Comment (twsouthwick) [inline src/Microsoft.Fx.Portability/TargetMapper.cs]: Same as above - can we combine with the declaration and use a `using`?

File: src/Microsoft.Fx.Portability/TargetMapper.cs [hunk 3]
```diff
@@ -149,6 +152,11 @@ private void Load(Stream stream, string path)
 
                 throw new TargetMapperException(message, e);
             }
+            finally
```
Comment (twsouthwick) [inline src/Microsoft.Fx.Portability/TargetMapper.cs]: If we can use `usings` this finally block an be removed

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Pull request merged**